tests/test_w3tagb.f90

```diff
@@ -11,5 +13,5 @@ program test_w3tagb
 
   ! prints information
   call w3tagb(prog, year, julian_day, hundreths_of_julian_day, org)
-  print *, "SUCCESS"
+  print *, "test_w3tagb success"
```

Comment:
No leave it SUCCESS, because that's what all the other tests are using. It makes it easy to spot that it worked...